arch/rv32i/src/pmp.rs

```diff
@@ -154,6 +160,8 @@ impl PMPConfig {
             // As we use the PMP TOR setup we only support half the number
             // of regions as hardware supports
             total_regions: pmp_regions / 2,
+            is_dirty: Cell::new(true),
+            last_configured_for: MapCell::empty(),
```

Comment:
Could you maybe drop an empty line before `is_dirty`, otherwise it looks like the comment applies to the block of code and not just `is_dirty`.

Reply:
Done.